proto/import_sstpb.proto

```diff
@@ -97,3 +101,28 @@ message CompactRequest {
 
 message CompactResponse {
 }
+
+message DownloadRequest {
+    // The SST meta used to identify the downloaded file.
+    // Must be the same among all nodes in the same Raft group.
+    // Note: the "range" and "cf_name" fields are ignored in this request.
+    SSTMeta sst = 2 [(gogoproto.nullable) = false];
+
+    // The URL of the external storage to fetch the folder containing SST file.
+    string url = 8;
+    // The file name of the SST file.
+    string name = 9;
+
+    // Performs a key prefix rewrite after downloading the SST file.
+    // All keys in the SST will be rewritten as:
+    //
+    //  new_key = new_key_prefix + old_key[old_key_prefix_length..]
+    uint32 old_key_prefix_length = 10;
+    bytes new_key_prefix = 11;
+
+    // The download speed limit (bytes/second). Set to 0 for unlimited speed.
+    uint64 speed_limit = 12;
+}
+
+message DownloadResponse {
```

Comment:
How does client know whether the RPC is succeed?

Reply:
If the RPC failed the gRPC response should be an error.

```go
	Download(ctx context.Context, in *DownloadRequest, opts ...grpc.CallOption) (*DownloadResponse, error)
	//                                                                        this would be non-nil ^~~~~
```

Reply:
I am not sure if it is enough to express errors like EpochNotMatch/RegionNotFound...

Reply:
@overvenus This request only copies a file from external_storage into local storage (plus key rewrite), and won't interact with PD. So `EpochNotMatch` and `RegionNotFound` should never happen.

Reply:
LGTM